src/events/ResultListEvents.ts

```diff
@@ -18,6 +18,18 @@ export interface IDisplayedNewResultEventArgs {
   item: HTMLElement;
 }
 
+/**
+ * The `IDisplayedNewResultsEventArgs` interface describes the object that all
+ * [`newResultsDisplayed`]{@link ResultListEvents.newResultsDisplayed} event handlers receive as an argument.
+ */
+export interface IDisplayedNewResultsEventArgs {
+  /**
+   * Whether the results are being displayed in a [`ResultList`]{@link ResultList} with infinte scroll enabled.
+   * See [`enableInfiniteScroll`]{@link ResultList.options.enableInfiniteScroll} option).
+   */
+  infiniteScrolling: boolean;
```

Comment:
I would remove this. Instead, to know if the results are currently being displayed in infinite scrolling, you can use the `ResultListUtils.isInfiniteScrollEnabled` function.

Reply:
That was my first try, but importing `ResultListUtils` was causing webpack errors -seemed like circular references-, which led me to believe that an event was a better way to know if infinite scrolling was enabled, instead of having a hard dependency on `ResultListUtils`/`ResultList`. Also, I saw `ResultListUtils. isInfiniteScrollEnabled ` is being used to hide the Pager and other components if all result lists have `enableInfiniteScroll` set to true, and in that case I also think that could be a bit easier with an event arg instead of looking in all of the result lists instances.